rpm/lua-rex.spec

```diff
@@ -0,0 +1,257 @@
+Name:           lrexlib
+Version:        2.9.0
+Release:        1%{?dist}
+Summary:        Regular expression handling library for Lua
+
+Group:          Development/Libraries
+License:        MIT
+URL:            http://lrexlib.luaforge.net/
+Source0:        lrexlib-%{version}.tar.gz
+
+%description
+Lrexlib are bindings of six regular expression library APIs (POSIX, PCRE, PCRE2,
+Oniguruma, Tre and GNU) to Lua.
+
+%package pcre
+Summary: Lua binding of PCRE library
+BuildRequires:  pcre-devel
+BuildRequires:  pkgconfig
```

Comment:
Common requires can be moved out of %package, I guess. I don’t insist, though.

Reply:
No, they can't.  I tried to move them out and got an error (for example) `tarantool: command not found` (we use tarantool during build to create actual rockspecs)